lib/trace-writer.js

```diff
@@ -68,32 +126,10 @@ TraceWriter.prototype.writeSpan = function(spanData) {
       spanData.trace.spans[i].close();
     }
   }
-  if (this.config_.hostname) {
-    spanData.addLabel(traceLabels.GCE_HOSTNAME, this.config_.hostname);
-  }
-  if (this.config_.instanceId) {
-    spanData.addLabel(traceLabels.GCE_INSTANCE_ID, this.config_.instanceId);
-  }
-
-  var moduleName = this.config_.gae_module_name || this.config_.hostname;
-  if (moduleName) {
-    spanData.addLabel(traceLabels.GAE_MODULE_NAME, moduleName);
-  }
-  if (this.config_.gae_module_version) {
-    spanData.addLabel(traceLabels.GAE_MODULE_VERSION,
-      this.config_.gae_module_version);
-
-    if (this.config_.gae_minor_version) {
-      var versionLabel = '';
-      if (moduleName !== 'default') {
-        versionLabel = moduleName + ':';
-      }
-      versionLabel += this.config_.gae_module_version + '.' +
-          this.config_.gae_minor_version;
-      spanData.addLabel(traceLabels.GAE_VERSION, versionLabel);
-    }
+  // Copy properties from the default labels.
+  for (var k in this.defaultLabels_) {
+    spanData.addLabel(k, this.defaultLabels_[k]);
```

Comment:
This is minor but it might be nice to initialize span data with default labels as opposed to copying them in one by one for every span.